packer/builder/azure/arm/config_test.go

```diff
@@ -201,3 +220,15 @@ func getPackerConfiguration() interface{} {
 
 	return config
 }
+
+func getPackerCommunicatorConfiguration() interface{} {
+	var doc = `{
+		"ssh_host": "packer.ssh",
+		"winrm_host": "packer.winrm"
```

Comment:
these are maybe 'weird' examples, since the host is determined at runtime anyway. Maybe leave a better trail for future coders and test something like ssh timeout?